### DISCUSSION TITLE
Add light theme

### DISCUSSION THREAD

**Discussion by k3davis** [directus/directus#5704] [Feature Requests]
Please consider adding a light or higher contrast theme to the administration UI. I'm excited about this project and it addresses several problems I was having with strapi - thanks for your efforts.

**Comment by benhaynes** [comment]
We do have a light theme... do you think it should be lighter? 🤔

Also, once we more formally add theme support, we'll definitely have a high contrast option.

**Reply by k3davis** [reply]
So sorry! I just overlooked the setting. Glad formal theme support is intended in the future though.
